src/docs/introduction/start-customizing/index.md

```diff
@@ -0,0 +1,149 @@
+  some steps. Learn what to take into consideration when starting custom
+  development
+published: true
+---
+
+When you start customizing your marketplace with code, you may want to
+continue working on the same web template that powers your hosted
+marketplace, or you may want to develop a fully custom application. Both
+options are possible, and there are some considerations for both
+approaches.
+
+## General custom development guidelines
+
+Whichever custom development approach you choose, here are some key
+points that apply to all custom development.
+
+### Get familiar with the Sharetribe APIs
+
+Your marketplace basic functions are available through the **Marketplace
+API**.
+
+Your no-code configurations are available through the public **Asset
+Delivery API**.
+
+Integration and analytics capabilities are available server-side through
+**Integration API**.
+
+Authentication to Marketplace API and Integration API happens through
+the **Authentication API**.
+
+Access to the Sharetribe APIs in the live environment is available in
+the Extend plan.
+
+The Marketplace API, Asset Delivery API, and Integration API are also
+accessible through JavaScript SDKs.
+
+- [Read more about Marketplace API and Integration API](/concepts/marketplace-api-integration-api/)
+- [Read more about Asset Delivery API](/references/assets/)
+- [Read more about Authentication API](/concepts/authentication-api/)
+- [Read more about Sharetribe JavaScript SDKs](/concepts/js-sdk/)
+- [Read more about Sharetribe pricing](https://www.sharetribe.com/pricing/)
+
+### Get familiar with Sharetribe environments
+
+Your marketplace has three environments – Test, Dev, and (eventually)
+Live. We recommend that you only do development work in Dev, and keep
+Test and Live identical in terms of code.
+
+Each marketplace environment connects to the corresponding client app
+through a Marketplace API application, i.e. a client id and a client
+secret. This means that when you call the Marketplace API or Asset
+Delivery API, the correct marketplace environment is determined by the
+client id set in your app.
+
+- [Read more about Sharetribe environments](/concepts/sharetribe-environments/)
+- [Read more about Sharetribe applications](/concepts/applications/)
+
+### Get familiar with hosting your application
+
+When you custom develop your marketplace, you also need to host your own
+application. When starting your custom development, it is enough to host
+your custom codebase for the Dev environment. When you want to publish
+your code, you will also need to host a client application for the Test
+and Live environments.
+
+When you have a hosted application set up for your custom client, you
+can change the Marketplace URL in your Console > Build > General view to
+point to the URL of your custom client. That way, the Sharetribe backend
+uses the correct URL for e.g. adding links to emails sent through the
+Sharetribe backend.
```

Comment:
1. One potential improvement could be explaining why and how you can benefit from a hosted version of your dev environment. You can also achieve most development tasks without hosting your dev environment at all. In addition, it's also not strictly necessary to host a test environment either. 

2. I think `custom client application` or similiar might work better than `custom codebase` in 
> it is enough to host your custom codebase for the Dev environment. 

I picture hosting a custom codebase referring more to source code management 🤔

3. The phrasing `hosted application set up for your custom client` is a bit confusing:
> When you have a hosted application set up for your custom client, you
> can change the Marketplace URL in your Console > Build > General view to
> point to the URL of your custom client.

 Would something like this work:

> Once you have deployed your marketplace application to a hosting service, you can...

Reply:
1. I agree that it would make sense to explain why it's (in my opinion) necessary to host your dev and test environments:
- hosted dev environment allows acceptance testing by e.g. the non-technical marketplace founder who wants to see how the features work
- hosted test environment allows the founder to continue making no-code changes on top of the custom dev application so they reliably know how their changes show up in Live

For a technical founder, they might get away with just developing their marketplace on localhost before going live, if they are doing all their acceptance testing locally as well, but after going live it's really hard to make any no-code changes in Test if it's not linked to the hosted client as well.

2. and 3. agreed, will make the updates!